app/services/validators/field-validator.js

```diff
@@ -0,0 +1,55 @@
+const validator = require('validator')
+const ERROR_MESSAGES = require('./validation-error-messages')
+
+class FieldValidator {
+  /**
+   * Build a validator for validating fields.
+   * @param data A single element to validate.
+   * @param fieldName The name of of the HTML element to link the error message to.
+   * @param errors An instance of the ErrorHandler class.
+   */
+  constructor (data, fieldName, errors) {
+    this.data = data
+    this.fieldName = fieldName
+    this.errors = errors
+  }
+
+  isRequired (specificMessage) {
+    var message = (!specificMessage) ? ERROR_MESSAGES.getIsRequiredMessage : specificMessage
+    if (!this.data) {
+      this.errors.add(this.fieldName, message)
+    } else if (this.data === 'select') {
```

Comment:
Is this check needed anymore now the value isn't being set to `select` in the reduction route?